src/web/components/ClickToView.tsx

```diff
@@ -137,19 +143,19 @@ export const ClickToView = ({
 				className={css`
 					width: 100%;
 					background: ${background.secondary};
-					border: 1px solid ${border.primary};
+					border: 1px solid ${border.secondary};
 					display: flex;
 					flex-direction: column;
 					justify-content: space-between;
-					padding: ${space[3]}px;
-					margin-bottom: 8px;
+					padding: 6px 25px 12px;
```

Comment:
`space` is essentially a base of 4 so these hard coded padding values could be replaced with, `space.1 space.6 space 3

Same for the other numbers

Reply:
I guess I was trying to do a pixel perfect version of the figma, using the space would not be identical.

I happy to 'interpret' the design if thats normally acceptable. 

What do you think?

Reply:
Ideally, the designs should be using base 4px - this is 'the official' [spacing system](https://theguardian.design/2a1e5182b/p/449bd5-space) but that extra pixel 'tweak' is very tempting!

The `space` tokens are, loosely, designed to prevent this. That said, we do sometimes deviate but normally only because things need to line up or for other concrete reasons. If it's a matter of subjectivity, I would say the design system should win.

Reply:
Ive left the 6px in and changed the rest to use the 'space' values.